project/api/models.py

```diff
@@ -7,42 +7,42 @@
 from colorfield.fields import ColorField
 
 # Create your models here.
-class Nation(models.Model):
-    """
-    Cultural/governmental entity. Serves as foreign key for most Territories
-    """
+class Entity(models.Model):
     name = models.TextField(max_length=100,
                             help_text="Canonical name, should not include any epithets, must be unique",
                             unique=True)
     url_id = models.SlugField(max_length=75,
-                              help_text="Identifier used to lookup nations in the URL, "
+                              help_text="Identifier used to lookup PoliticalEntitys in the URL, "
                                         "should be kept short and must be unique",
                               unique=True)
-    color = ColorField(help_text="Color to display on map",
-                       unique=True,
-                       null=True,
-                       blank=True)
-    history = HistoricalRecords()
-
-    ## Flavor fields
-
-    # required fields
     references = ArrayField(
         models.TextField(max_length=150),
     )
-
-    # optional fields
-    aliases = ArrayField(
-        models.TextField(max_length=100),
-        help_text="Alternative names this state may be known by",
+    links = ArrayField(
```

Comment:
The idea was that links would be additional flavor containing further information about the Entity. Wikipedia could be one, Britannica, maybe history.com, etc. Limiting the link to be a wp article that's just a stub or us missing other prolific information isn't particularly helpful, what do you think?

Reply:
Hmm, in this case let's maybe remove links altogether?

On the frontend we will show all the references, and if we show them as links, then I don't see the point of having another `links` field. I'm thinking like this Denmark link: https://en.wikipedia.org/wiki/Napoleonic_Wars#cite_note-42, something like that pointing to history.com or Britannica.

Or the second idea is to have a derivative column called `links`: i.e. the backend will parse all references that are links, and present those to the frontend.

Reply:
I'm confused by the second idea, how is that different from the current field?
Otherwise I was thinking that references would cite only information regarding where the information stored in our database was referenced from, and flavor fields that are relevant but not places that we cite would be in `links`.

Reply:
Yes, I thought the same when I first saw `links` in your previous PR, and sounded fine to me. But do you have an example where a link concerning a political entity would go inside `references` and not `links` (and conversely?)

The only reason I'm raising this issue is that I don't want the user to input the same link twice in the PoliticalEntity creation form.

My second idea was: the user only inputs references (some are books, some are URLS). The backend filters the one that are URLs, and save those into the link column.

Reply:
If we had territory mapping the French Wars of Religion, it may be helpful to provide some background info regarding the reasons for the conflict and rising religious tensions in `links`, but as its not directly related to the territories that any of the involved parties controlled during this time or other in formation about the Political Entity it wouldn't be appropriate to put in `references`. 

Also, I think we shouldn't restrict people to one site (wikipedia) for further information, other sites may have more complete or relevant data.

Reply:
I see what you mean.

I still feel like it might be slightly confusing for contributors:
- references are directly talking about the political entity
- and links are related to reasons, context and other content around the political entity, but not talking directly about it.
So wikipedia would go in references and not in links. Btw I totally agree we shouldn't limit to wikipedia.

Let's close this discussions and go your way: links + references, it's always better to have too much data than not enough. I'll add this on the frontend, and come up with an explanatory text for contributors.

Reply:
I just remembered Wikipedia oftentimes distinguishes between "Further reading" and "References", see https://en.wikipedia.org/wiki/Wikipedia:Further_reading#Relation_to_reference_sections. A similar system is what I had in mind.

Reply:
OK! Yeah, you convinced me, let's go this way 👍 .

Reply:
Lol just wanted to link an example 😅